deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -35,7 +35,8 @@ metadata:
                             "key": "kubernetes.io/arch",
                             "operator": "In",
                             "values": [
-                              "amd64"
+                              "amd64",
+                              "s390x"
```

Comment:
do we need to support Z for openshift in this release? AFAIK, ocp on Z is different versioning with x86.

Reply:
We plan to, yes. whether if they are different or not, we will test both of them.